tests/app/v2/govuk_alerts/test_get_broadcasts.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from flask import current_app, json
+
+from app.models import BROADCAST_TYPE
+from notifications_python_client.authentication import create_jwt_token
+from tests.app.db import create_broadcast_message, create_template
+
+
+def test_get_all_broadcasts_returns_list_of_broadcasts_and_200(
+    client, sample_broadcast_service
+):
+    template_1 = create_template(sample_broadcast_service, BROADCAST_TYPE)
+
+    broadcast_message_1 = create_broadcast_message(
+        template_1,
+        starts_at=datetime(2021, 6, 15, 12, 0, 0),
+        status='cancelled')
+
+    broadcast_message_2 = create_broadcast_message(
+        template_1,
+        starts_at=datetime(2021, 6, 22, 12, 0, 0),
+        status='broadcasting')
+
+    govuk_alerts_jwt_client_id = current_app.config['GOVUK_ALERTS_CLIENT_ID']
```

Comment:
Now https://github.com/alphagov/notifications-api/pull/3303 is in you can just do [this](https://github.com/alphagov/notifications-api/pull/3303/commits/09e3ba6836df0a930392a81dd3569f10ab4d04b6#diff-a705d52d5175ca6300ff24be4c10207e0c197282c3f89d39435b8fb1e46506a3R83).

Reply:
cool stuff, I pulled in those changes now :3